client/src/components/VerifyInfo/index.stories.tsx

```diff
@@ -8,10 +8,10 @@ export default {
 
 const baseProps = {};
 
-export const Base = () => <VerifyInfo {...baseProps} />;
+export const Base = () => <VerifyInfo {...baseProps} onVerify={() => {}} />;
```

Comment:
We can use `action('onVerify')` for the `onVerify` prop of these stories, imported from `@storybook/addon-actions` I think.

That way we'll get a little action notification in the addons panel 🚀